### PR TITLE
Robots-101 team supervisor post

### PR DESCRIPTION
**Pull request by antoinepetty**

### CODE REVIEW

**Comment by antoinepetty** [comment]
Preview
image

**Comment by PeterJCLaw** [comment]
Does this fix https://github.com/srobo/docs/issues/323?
Should this content be duplicated into the team-admin section in the docs?

**Comment by antoinepetty** [comment]
> Does this fix [srobo/docs#323](https://github.com/srobo/docs/issues/323)? Should this content be duplicated into the team-admin section in the docs?

We're intending to run this 101 series and then port the important info to the docs at a later date so I would leave that issue open as a reminder

File: _posts/2022-10-07-101-so-youre-a-team-supervisor.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+So, you’re running a team. Here’s a little bit of info on what to expect from Student Robotics, and your responsibilities as a team supervisor.
+
+As a team supervisor, your role is to guide the competitors through the journey of building a robot. You’ll be there to point them in the right direction when they get stuck and resolve any issues they run into. We encourage team supervisors to limit their involvement with the robot design/building process so that the finished contraptions are 100% student-built which competitors find very rewarding.
+
+You’ll be our point of contact with the team. If you have any questions during the year, just email [teams@studentrobotics.org](mailto:teams@studentrobotics.org). We’ll also be sending you emails every month or so with important information such as:
+- Details of Tech Days where we provide a space for teams to come together and get direct help from our dedicated volunteers
+- Software updates for our kit
+- Information about the competition in April
+
+We aim to host our Kickstart event and Tech Days in multiple locations to make it more convenient for you to travel. However, you will still need to arrange to travel to these places.
```
Comment (samjmartin04) [inline _posts/2022-10-07-101-so-youre-a-team-supervisor.md:17]: I suppose hosting kickstarts in multiple locations may be a bit out of date at this point, not that it matters too much

**Review by samjmartin04 — APPROVED** [review]

**Pull request merged**